### ISSUE TITLE
最新版obsidian 不兼容

### ISSUE THREAD

**Issue by BonjourL** [akaalias/obsidian-extract-pdf#22]
最新版obsidian 不兼容，打开这个插件，在obsidian中pdf 就显示不出来

**Comment by raywang2018** [comment]
The latest version of obsidian is not compatible with this plug-in, when open this plug-in, the pdf in obsidian will not be displayed

**Comment by iSaniiro** [comment]
Exactly. Got the same issue here. However the extractor still works. Seems like it has something to do with the viewer.

**Comment by Wattskemov** [comment]
Still can't work under Obsidian 1.4.16. When enable this plugin, Obsidian can't show pdf. And if turn this plugin off, then Obisidan work well with pdf file. Is there any update plan here?
